package-lock.json

```diff
@@ -101,6 +101,9 @@
         "webpack-cli": "^4.10.0",
         "webpack-dev-server": "^4.9.3",
         "webpack-merge": "^5.8.0"
+      },
+      "engines": {
+        "node": "^20"
```

Comment:
I noticed that locally running `npm i` is changing this line to `"node": ">= 20"`.